packages/integration-sdk/README.md

```diff
@@ -5,6 +5,9 @@ Note: This package has been been deprecated and should no longer be used.
 The functionality provided by this package has been broken up into multiple
 smaller packages.
 
-If you are developing an integration, you can use the
-`@jupiterone/integration-sdk-core` and `@jupiterone/integration-sdk-cli`
-packages.
+This is now a shell package used to store historical information about the older
+`@jupiterone/integration-sdk` and also track changes made to the the
```

Comment:
dammit just notice the double `the`. Mind if I fix that after the merge so I don't have to delete all the tags?